src/ecmult_impl.h

```diff
@@ -696,7 +702,9 @@ static int secp256k1_ecmult_strauss_batch_single(const secp256k1_callback* error
 }
 
 static size_t secp256k1_strauss_max_points(const secp256k1_callback* error_callback, secp256k1_scratch *scratch) {
-    return secp256k1_scratch_max_allocation(error_callback, scratch, STRAUSS_SCRATCH_OBJECTS) / secp256k1_strauss_scratch_size(1);
+    /* Call max_allocation with 0 objects because we've already accounted for
+     * alignment in strauss_scratch_size. */
+    return secp256k1_scratch_max_allocation(error_callback, scratch, 0) / secp256k1_strauss_scratch_size(1);
```

Comment:
... and wasn't the previous version (without changes in `strauss_scratch_size`) more precise?
We need to round up to the alignment only once per array (e.g., once for the scalars array). 

In the proposed revision, I think we overestimate the required padding a lot because we still call `strauss_scratch_size(1)` here but this has padding now.

Reply:
This is correct - sorry I overlooked this. Will add fix and test.

Reply:
fixed